### PR TITLE
CUBE: refactor code and add meta data

### LINKED ISSUE #9308: Make remaining CUBE code review changes
See remaining unresolved comments [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/8990)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Fix typo in nav
- Revert to using `getAttribute` not `nextElementSibling` to be more specific when targeting share menu
- Make the `.is-faded` styling a placeholder which is then extended instead of extending a class
- Add meta descriptions 
- Add `js-` prefix to class

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube and http://0.0.0.0:8001/cube/microcerts
- Check (update?) meta descriptions and titles
- Check points raised in this this [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/8990) have been addressed
- Check refactored code

## Issue / Card

Fixes #9308

### CODE REVIEW

**Comment by bethcollins92** [comment]
@anthonydillon FYI remaining refactoring and changes to CUBE in this PR

File: templates/cube/index.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {% extends "cube/base_cube.html" %}
 
-{% block title %}CUBE{% endblock %}
+{% block title %}The Certified Ubuntu Engineer (CUBE){% endblock %}
 
-{% block meta_description %}This is the CUBE landing page{% endblock meta_description %}
+{% block meta_description %}The Certified Ubuntu Engineer (CUBE) is a professional endorsement indicating mastery of Ubuntu. It consists of 15 microcertifications to complete.{% endblock meta_description %}
```
Comment (pmahnke) [inline templates/cube/index.html:5]:
```suggestion
{% block meta_description %}The Certified Ubuntu Engineer (CUBE) is a professional endorsement indicating mastery of Ubuntu. It consists of 15 microcertifications.{% endblock meta_description %}
```

File: templates/cube/index.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {% extends "cube/base_cube.html" %}
 
-{% block title %}CUBE{% endblock %}
+{% block title %}The Certified Ubuntu Engineer (CUBE){% endblock %}
```
Comment (pmahnke) [inline templates/cube/index.html:3]:
```suggestion
{% block title %}Certified Ubuntu Engineer (CUBE){% endblock %}
```

File: templates/cube/microcerts.html [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {% extends "cube/base_cube.html" %}
 
-{% block title %} Microcerts | CUBE {% endblock %}
+{% block title %} Microcertifications | CUBE {% endblock %}
 
-{% block meta_description %}This is the CUBE Microcerts page{% endblock meta_description %}
+{% block meta_description %}The path to certification is clear and convenient. Complete 15 microcerts to attain CUBE.{% endblock meta_description %}
```
Comment (pmahnke) [inline templates/cube/microcerts.html:5]:
```suggestion
{% block meta_description %}Follow the path to certification - complete the 15 microcerts to attain CUBE.{% endblock meta_description %}
```

**Review by pmahnke — APPROVED** [review]
A few text suggestions.

**Pull request merged**